align/config/basic_template.sp

```diff
@@ -1,10 +1,10 @@
 .subckt NMOS_4T  D G S B
-* @: Generator(name='mos')
+* @: Generator(name='MOS')
```

Comment:
What is the purpose of capitalizing the name? Why not capitalize this in the constraint schema instead?
Also, the compiler still hard codes `MOSGenerator` despite this statement. Let's get rid of this hard-coding in the next PR as well.

Reply:
Didn't do it in schema considering some generator names are case sensitive? If you think its fine to convert generator names to upper case, I can implement that.

Reply:
Compiler does not hard code to ```MOSGenerator``` but its a string based match ```if MOS in MOSGenerator" which maps it to ```MOSGenerator```. We can convert it to an exact match.

Reply:
You are right, not in compiler. MOSGenerator is hard-coded in align/primitive and align/pnr. This can be addressed in another PR. No changes needed.